pkg/networking/cilium/client.go

```diff
@@ -25,43 +25,44 @@ type Client interface {
 	RolloutRestartDaemonSet(ctx context.Context, name, namespace, kubeconfig string) error
 }
 
-type retrierClient struct {
+// RetrierClient wraps basic kubernetes API operations around a retrier
+type RetrierClient struct {
```

Comment:
I just made this public to not have tests in the `cilium` package. There were circular imports because of the mocks package